gooey/gui/containers/application.py

```diff
@@ -121,9 +121,13 @@ def onComplete(self, *args, **kwargs):
         """
         with transactUI(self):
             if self.clientRunner.was_success():
-                self.showSuccess()
-                if self.buildSpec.get('show_success_modal', True):
-                    wx.CallAfter(modals.showSuccess)
+                if self.buildSpec.get('return_to_config', False):
+                    self.showSettings()
+                    self.footer.progress_bar.Show(False)

```

Comment:
I'd put this line down in the showSettings() method. Other than that, lgtm.

Reply:
Do you mean, move the `self.footer.progress_bar.Show(False)` statement to the `showSettings()` method?